src/page-overview/OverviewPage.tsx

```diff
@@ -1,3 +1,4 @@
+import { isIE } from "react-device-detect";
```

Comment:
I think it would be better if you just reverted the IE detection commit to get rid of these changes? that way we're not looking at two different features here

Reply:
Agreed. For the record, @AhmedMugtaba, we want each PR to be focused on a single task, and we don't want to include work that we've already decided we're no longer pursuing. You should create a new branch each time you begin work on a new feature or task, as opposed to one large branch that you work on across all of your features.

Reply:
Got it, will update.